Formula/gcc-arm-embedded.rb

```diff
@@ -0,0 +1,43 @@
+class GccArmEmbedded < Formula
+  desc "GNU Embedded Toolchain for Arm Cortex-M/R processors"
+  homepage "https://developer.arm.com/open-source/gnu-toolchain/gnu-rm"
+  url "https://developer.arm.com/-/media/Files/downloads/gnu-rm/9-2019q4/RC2.1/src%202.0/gcc-arm-none-eabi-9-2019-q4-major-src.tar.bz2"
+  version "9-2019-q4-major"
+  sha256 "f162a655f222319f75862d7aba9ff8a4a86f752392e4f3c5d9ef2ee8bc13be58"
+
+  # build-prerequisites fails without this, when ./configure is run from zlib it raises an error,
+  # the zlib log shows the problem is: "The build tool has reset ENV; --env=std required."
+  env :std
+
+  depends_on "gnu-tar" => :build
+  depends_on :xcode => ["10.0", :build]
+
+  def install
+    inreplace "build-common.sh", "JOBS=1", "JOBS=`sysctl -n hw.ncpu`"
+
+    system "./install-sources.sh", "--skip_steps=mingw32"
```

Comment:
We should install such sources as resources, not download them from a script.

Reply:
I hear what you are saying, however this is what the directions for building this say to do.

This is an excerpt from their instructions which come included with tarball:

```
# Prepare source codes
$ cd ~/mac-build
$ tar xjf gcc-arm-none-eabi-9-2019-q4-major-src.tar.bz2 $ cd ./gcc-arm-none-eabi-9-2019-q4-major
$ ./install-sources.sh
# Start building the toolchain. $ ./build-prerequisites.sh
$ ./build-toolchain.sh
```

Some of the sources are not even downloaded, they just come packaged in the source tarball.